jenkinsfiles/Windows_Check.groovy

```diff
@@ -105,7 +105,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```

Comment:
TIMESTAMP is correct, but DATESTEMP should be changed to DATESTAMP to be consistent.